### PR TITLE
ChatHeader Component

### LINKED ISSUE #71: Top Chat Bar Component
**Describe the task that needs to be done.**

Create a chat bar component showing profile badge, buddy name, active status if online and a button to call buddy.
Design on Figma:
Chatbar

**Describe how a solution to your proposed task might look like (and any alternatives considered).**
This component will make use of the existing profile badge component.

**Notes**

### PR DESCRIPTION
**Pull request by colaMeowSY**
## Description
Implement the chatHeader component using the profile badge component. Followed design on Figma.

Closes #71 

## How has this been tested?
Simple snapshot tests and react tests to see if the component is correctly rendered and expected text appear.

## Screenshots
*Provide screenshots of what has been implemented. Leave blank if not applicable*
[Updated]
Buddy is online:

online

Buddy is offline:

offline

## Checklist
*(Leave blank if not applicable)*

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/ChatHeader/ChatHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <div className={styles.userDetails}>
+        <ProfileBadge active={active} name={name} />
+        <div className={styles.detailsText}>
+          <p
+            className={`${styles.headerText} ${styles.buddyName} ${
+              active ? styles.onlinePadding : styles.offlinePadding
+            }`}>
+            {name}
+          </p>
+          <p className={`${styles.headerText} ${styles.status}`}>{active ? 'Active now' : ' '}</p>
```
Comment (scorpionknifes) [inline src/components/ChatHeader/ChatHeader.jsx:25]: use <Typography> from MUI for material UI text style

**Review by scorpionknifes — CHANGES_REQUESTED** [review]
Looks good,

just need to use mui font style

File: src/components/ChatHeader/ChatHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import styles from './ChatHeader.module.css';
+
+/**
+ * ChatHeader component displays the avatar, name and a status of the buddy the current
+ * user in the current conversation.
+ * A call button will allow user to give a call to this buddy.
+ */
+function ChatHeader({ active, name, callBuddy }) {
+  return (
+    <div className={styles.chatHeader}>
```
Comment (KirstyGong) [inline src/components/ChatHeader/ChatHeader.jsx:15]: You can try use more mui components next time. It will save u lots time from styling.

**Review by DanilLinkov — APPROVED** [review]
Looks good great work

**Review by scorpionknifes — APPROVED** [review]
LGTM

**Pull request merged**